runway_model.py

```diff
@@ -31,45 +31,38 @@ def setup(opts):
     return model
 
 
-inputs = {"input_file" : runway.image}
-outputs = {"output_file" : runway.image}
+inputs = {"input_image" : runway.image}
+outputs = {"output_image" : runway.image}
 
 
-@runway.command("Segment Humans", inputs=inputs, outputs=outputs, description="Segments Humans")
+@runway.command("segment_humans", inputs=inputs, outputs=outputs, description="Segments Humans")
```

Comment:
Since the command names are converted into HTTP endpoints, better to avoid using spaces (we will enforce that on the SDK side as well in a future update)